### PR TITLE
Move files_versions to webpack

### PR DESCRIPTION
**Pull request by rullzer**
My first attempt of moving basic JS code to webpack.

* [x] Move JS over to bundle
* [x] Move CSS over to bundle

Future work:
- handlebars loader to do magic handlebars stuff on webpack bundling
- ~~at some point move to global webpack bundle thingy~~ :heavy_check_mark: 
- maybe move whole component to vue

To check:
- [x] Does it still work with richdocument sidebar hacks

### CODE REVIEW

**Comment by rullzer** [comment]
Full bundle is 18kb. Which is approx the same as the merged.json one (but now with the CSS).

It can now be served statically by the webservers! :tada:
And the proper map file helps a lot as well :smile:

File: apps/files_versions/package.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
+{
+  "name": "files_versions",
+  "version": "1.0.0",
+  "description": "files versions for nextcloud",
+  "private": true,
+  "dependencies": {
+    "terser": "^3.14.1"
```
Comment (rullzer) [inline apps/files_versions/package.json]: Temp hack because else it does :boom:

**Comment by rullzer** [comment]
> handlebars loader to do magic handlebars stuff on webpack bundling

I checked this. if we do that now if basically includes the handlebar runtime. Making the whole bundle around 30kb

File: apps/files_versions/src/webpack.dev.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+const merge = require('webpack-merge');
+const common = require('./webpack.common.js');
+
+module.exports = merge(common, {
+  mode: 'development',
+  devServer: {
```
Comment (skjnldsv) [inline apps/files_versions/src/webpack.dev.js]: Not needed ;)

**Comment by rullzer** [comment]
Ready to get in!

**Review by ChristophWurst — APPROVED** [review]
Looks good!

**Review by MorrisJobke — APPROVED** [review]
Tested and still works :+1:

**Comment by MorrisJobke** [comment]
JSUnit failures: https://drone.nextcloud.com/nextcloud/server/15786/265

**Pull request merged**